src/main/kotlin/net/maizegenetics/phgv2/brapi/model/ServerInfoResponse.kt

```diff
@@ -1,6 +1,7 @@
 package net.maizegenetics.phgv2.brapi.model
 
 import kotlinx.serialization.Serializable
+import model.Context
```

Comment:
Also not used?

Reply:
Also used in the class def - am I missing something?

```
data class ServerInfoResponse(
    val metadata: Metadata,
    val result: ServerInfo,
    val _atContext: Context? = null
) {
```

Reply:
Ok, but that was already in the data class? Even before adding the import?

Reply:
@tcasstevens It was.  When I look at history of this file, the original version had the "val  _atContext: Context? = null" but not the import.  I don't know how this compiled.

When I remove this import, Context is not defined and I have choices.  I believe we need the one that lives in Brapi Model, so I copied it from the phg_webktor_server.

The phg_webktor_server versions of these files have the packages as "model" and they don't need this import.

But the phg_v2 version has a package of net.maizegenetics.phgv2.brapi.model and if I  don't add the import, then "Context" is not found.

Is there something different I should be doing?

Reply:
I think it's being found because they are in the same package.  model.Context shouldn't be in PHGv2. That is the package from phg_webktor_server